packages/button/package.json

```diff
@@ -99,7 +99,7 @@
         "@spectrum-web-components/shared": "1.4.0"
     },
     "devDependencies": {
-        "@spectrum-css/button": "14.1.1"
+        "@spectrum-css/button": "14.1.4"
```

Comment:
The latest (and the version currently being used on `main`) is 14.1.6, let's make sure this is up to date before merging.